app/web/src/newhotness/Explore.vue

```diff
@@ -124,14 +172,39 @@ const actionViewList = computed(
   () => actionViewListRaw.data.value?.actions ?? [],
 );
 
-const key = makeKey("ComponentList");
-const args = makeArgs("ComponentList");
-const componentList = useQuery<BifrostComponentList | null>({
-  queryKey: key,
-  queryFn: async () => await bifrost<BifrostComponentList>(args),
+const componentListRaw = useQuery<BifrostComponentList | null>({
+  queryKey: makeKey("ComponentList"),
+  queryFn: async () =>
+    await bifrost<BifrostComponentList>(makeArgs("ComponentList")),
 });
+const componentList = computed(
+  () => componentListRaw.data.value?.components ?? [],
+);
+
+const scrollRef = ref<HTMLDivElement>();
+
+const virtualizerOptions = computed(() => {
+  return {
+    count: componentList.value.length,
+    // `lanes` gives virtualizer a "second-dimension" (aka columns for vertical lists and rows for horizontal lists)
+    // https://tanstack.com/virtual/latest/docs/api/virtualizer#lanes
+    // Our grid is based on a minimum 250px width tile... so how many tiles can we fit?
+    // thats the value of `lanes`
+    lanes: Math.floor(scrollRef.value?.offsetWidth ?? 0 / 250),
```

Comment:
Clever

Reply:
we're very glad they have this `lanes` parameter... otherwise it would have gotten a lot uglier